src/App.test.tsx

```diff
@@ -3,9 +3,10 @@ import {render, screen} from "@testing-library/react";
 import App from "./App";
 
 test("renders learn react link", () => {
+  process.env.VITE_TITLE = "React Template development";
```

Comment:
Those kinds of definitions should probably go in a specific file. Jest support the use of a specific file to set up tests.